### PR TITLE
apptestai-test 0.0.2

### PR DESCRIPTION
**Pull request by forcemax**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/apptestai-test/0.0.2/step.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+toolkit:
+  go:
+    package_name: github.com/forcemax/bitrise-step-apptestai-test
+deps: {}
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- access_key: ci_support:4b55ec5999ea636b0aafb402816ac50b
```
Comment (koral--) [inline steps/apptestai-test/0.0.2/step.yml]: Key value must not be here.

File: steps/apptestai-test/0.0.2/step.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  opts:
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    title: apptest.ai access key (userid:accesskey).
+- project_id: 12389
```
Comment (koral--) [inline steps/apptestai-test/0.0.2/step.yml]: ditto

File: steps/apptestai-test/0.0.2/step.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  opts:
+    is_expand: true
+    is_required: true
+    title: apptest.ai project id.
+- binary_path: app/build/outputs/apk/release/app-prod-release.apk
```
Comment (koral--) [inline steps/apptestai-test/0.0.2/step.yml]: It should be `$BITRISE_APK_PATH` or an equivalent for IPA.

File: steps/apptestai-test/0.0.2/step.yml (hunk 1 above)
Comment (forcemax) [reply steps/apptestai-test/0.0.2/step.yml]: The value is an example value. Are you saying you want to remove the default value?

File: steps/apptestai-test/0.0.2/step.yml (hunk 2 above)
Comment (forcemax) [reply steps/apptestai-test/0.0.2/step.yml]: Same as above.

File: steps/apptestai-test/0.0.2/step.yml (hunk 1 above)
Comment (koral--) [reply steps/apptestai-test/0.0.2/step.yml]: Yes. Default values are not intended to be examples.
If there are many keys/ids in this service and users may be confused you can add an example in description.

**Comment by forcemax** [comment]
ok.
For access_key and project_id input, I will remove the default value, and binary_path input will change the default value to $BITRISE_APK_PATH.
Also, I will add a summary for the inputs and write down the example value.
I will request a new pull request to reflect this.

**Pull request closed without merging**